docs/gui.rst

```diff
@@ -74,39 +75,40 @@ Paint a window
 
     The data type of ``img`` must be one of:
 
-    * float32, clamped into [0, 1]
+    * ``float32``, range ``[0, 1]``
 
-    * float64, clamped into [0, 1]
+    * ``float64``, range ``[0, 1]``
 
-    * uint8, range [0, 255]
+    * ``uint8``, range ``[0, 255]``
 
-    * uint16, range [0, 65535]
+    * ``uint16``, range ``[0, 65535]``
 
-    * uint32, range [0, UINT_MAX]
+    * ``uint32``, range ``[0, 4294967295]``
 
 
 .. function:: gui.circle(pos, color = 0xFFFFFF, radius = 1)
 
     :parameter gui: (GUI) the window object
-    :parameter pos: (tuple of 2) the position of circle
-    :parameter color: (optional, RGB hex) color to fill the circle
-    :parameter radius: (optional, scalar) the radius of circle
+    :parameter pos: (tuple of 2) the position of the circle
+    :parameter color: (optional, RGB hex) the color to fill the circle
+    :parameter radius: (optional, scalar) the radius of the circle
 
     Draw a solid circle.
 
 
 .. function:: gui.circles(pos, color = 0xFFFFFF, radius = 1)
 
     :parameter gui: (GUI) the window object
-    :parameter pos: (np.array) the position of circles
-    :parameter color: (optional, RGB hex or np.array of uint32) color(s) to fill circles
-    :parameter radius: (optional, scalar) the radius of circle
+    :parameter pos: (np.array) the positions of the circles
+    :parameter color: (optional, RGB hex or np.array of uint32) the color(s) to fill the circles
+    :parameter radius: (optional, scalar) the radius (radii) of the circles
```

Comment:
```suggestion
    :parameter pos: (np.array) positions of the circles
    :parameter color: (optional, RGB hex or np.array of uint32) color(s) to fill the circles
    :parameter radius: (optional, scalar or np.array of uint32) the radius (or radii) of the circles
```
`scalar or np.array of uint32`, since you said `radii`..
Btw, do we have `radius(es)` in en? Not sure if all people know what this is..

Reply:
No, we don't really have radiuses. Probably there is a word like that in dictionaries but I never saw it being used.